### DISCUSSION TITLE
How to undo `add` or `re-add`

### DISCUSSION THREAD

**Discussion by leonasdev** [twpayne/chezmoi#3195] [Q&A]
like git's `git restore --staged .bashrc` but using chezmoi commandline

**Comment by twpayne** [comment]
You can use [`chezmoi git`](https://www.chezmoi.io/reference/commands/git/) to run git commands in the source directory, so you can do:

```console
$ chezmoi git restore --staged $(chezmoi source-path .bashrc)
```
